parametric/test_library_tracestats.py

```diff
@@ -255,7 +268,7 @@ def test_top_level_TS005(library_env, test_agent, test_library):
 
 @pytest.mark.skip_library("nodejs", "nodejs has not implemented stats computation yet")
 @enable_tracestats()
-def test_successes_errors_recorded_separately_TS006(library_env, test_agent, test_library):
+def test_successes_errors_recorded_separately_TS005(library_env, test_agent, test_library, test_server):
```

Comment:
Can we revert the changes to the test names? They're supposed to line up with the names here: https://docs.google.com/document/d/159HICPGmFQW1mW6sFHyNL0GKYPExQ6cWNHY31jec3mc/edit#heading=h.all329scnpmr

Reply:
Can do 👍